source/connect/stable-api.txt

```diff
@@ -0,0 +1,117 @@
+the API version changes, which can change the way these operations behave.
+Using the {+stable-api+} ensures consistent responses from the server and 
+provides long-term API stability for your application.
+
+The following sections describe how you can enable and customize {+stable-api+} for
+your MongoDB client. For more information about the {+stable-api+}, including a list of 
+the commands it supports, see :manual:`Stable API </reference/stable-api/>` in the
+{+mdb-server+} manual.
+
+Enable the {+stable-api+}
+-------------------------
+
+To enable the {+stable-api+}, perform the following steps:
+
+1. Construct a ``MongoDB\Driver\ServerApi`` object and pass the {+stable-api+}
+   version you want to use. Currently, the library supports only version 1.
+#. Construct a ``MongoDB\Client`` object. For the ``driverOptions`` parameter, pass an
+   array that contains the ``serverApi`` connection option. Set this option to the
+   ``MongoDB\Driver\ServerApi`` object you created in the previous step.
+
+The following code example shows how to specify {+stable-api+} version 1:
+
+.. literalinclude:: /includes/connect/stable-api.php
+   :language: php
+   :copyable: true
+   :start-after: // start-specify-v1
+   :end-before: // end-specify-v1
+   :emphasize-lines: 5-7, 9
+
+.. note::
+
+   After you create a ``MongoDB\Client`` instance with
+   a specified API version, all commands you run with the client use the specified
+   version. If you need to run commands using more than one version of the 
+   {+stable-api+}, create a new ``MongoDB\Client`` instance.
+
+.. _stable-api-options:
+
+Configure the {+stable-api+}
+------------------------
+
+The following table describes the parameters of the ``MongoDB\Driver\ServerApi`` constructor.
+You can use these parameters to customize the behavior of the {+stable-api+}.
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 25,75
+
+   * - Parameter
+     - Description
+
+   * - strict
+     - | **Optional**. When ``true``, if you call a command that isn't part of 
+         the declared API version, the library raises an exception.
+       |
+       | Default: ``false``
+
+   * -  deprecationErrors
+     - | **Optional**. When ``true``, if you call a command that is deprecated in the 
+         declared API version, the library raises an exception.
+       |
+       | Default: ``false``
+
+The following code example shows how you can use these parameters when constructing a 
+``MongoDB\Driver\ServerApi`` object:
+
+.. literalinclude:: /includes/connect/stable-api.php
+      :language: php
+      :copyable: true
+      :start-after: // start-stable-api-options
+      :end-before: // end-stable-api-options
+      :emphasize-lines: 5-7, 9
+
+API Documentation
+-----------------
+
+For more information about the ``MongoDB\Client`` class, see the following {+driver-short+}
+API documentation:
+
+- :ref:`MongoDB\Client <php-api-mongodbclient>`
```

Comment:
Same comment below about using `:phpclass:` instead. I'm actually not sure why it's necessary to reference this in a bullet instead of just hyperlinking ``MongoDB\Client``  in the above paragraph.

Reply:
that's the way we link to api docs on all pages, probably because many of them include multiple links.